src/com/pentalog/pentastagiu/homework2/bank/BankAccount.java

```diff
@@ -3,24 +3,23 @@
 public class BankAccount {
     private double moneyAmount;
 
-    public BankAccount(double p_moneyAmount) throws NotEnoughMoneyException {
-        if (p_moneyAmount < 10) {
-            throw new NotEnoughMoneyException("A bank account must be created with at least 5 dollars!");
+    public BankAccount(double moneyAmount) throws NotEnoughMoneyException {
+        if (moneyAmount < 10) {
+            throw new NotEnoughMoneyException("A bank account must be created with at least 10 dollars!");
         }
-        moneyAmount = p_moneyAmount;
-        System.out.println("A bank account with " + moneyAmount + " has been created!");
+        this.moneyAmount = moneyAmount;
+        System.out.println("A bank account with " + this.moneyAmount + " has been created!");
     }
 
     public void displaySold() {
         System.out.println("The account has " + moneyAmount);
     }
 
-    public void withdraw(Double value) {
-        if(value <= 0){
+    public void amountToWithdraw(Double value) {
```

Comment:
era ok inainte, cand era denumita metoda withdraw pt ca e ok sa exprime o actiune; schimbarea pe care o sugerasem eu se referea ca parametrul metodei sa fie redenumit, din value in amoutTowWithdraw fiidnca value e prea generic